src/app/signIn/signIn.tpl.html

```diff
@@ -5,9 +5,16 @@
         <div class="panel sign-in-panel">
           <div class="panel-body">
             <div class="row vdivide">
+              <div class="col-md-6 sign-in-section sign-in-hide-for-desktop">
```

Comment:
I basically duplicated the "New Partners" section, and added conditional classes that hides one and shows the other.  Ideally it would be great to have just one instance of this section and have conditional logic that flips the order of the sections... but this was what I could find.

Reply:
It sounded like they would be ok with rearranging the two on desktop as well during our last meeting.